test/fixtures/error-config/webpack.config.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+const path = require('path');
+
+module.exports = {
+  mode: 'development',
+  context: __dirname,
+  entry: './foo.js',
+  output: {
+    path: path.resolve(__dirname, `../../outputs/`),
```

Comment:
nits: Why did you use a template literal?

Reply:
Good catch, need setup eslint rule for this :+1: